test/main.sh

```diff
@@ -54,11 +54,20 @@ available_storage_backends() {
   local backend backends
 
   backends="dir"
-  for backend in btrfs lvm zfs; do
-    if which $backend >/dev/null 2>&1; then
-      backends="$backends $backend"
-    fi
-  done
+
+  if [ "${LXD_CEPH_CLUSTER:-}" ]; then
+    for backend in btrfs ceph lvm zfs; do
+      if which $backend >/dev/null 2>&1; then
+        backends="$backends $backend"
+      fi
+    done
+  else
+    for backend in btrfs lvm zfs; do
+      if which $backend >/dev/null 2>&1; then
+        backends="$backends $backend"
+      fi
+    done
+  fi
```

Comment:
Wouldn't this code be cleaner if we had a separate variable storing the list of all existing backends minus ceph?

Something like:
```bash
storage_backends="btrfs lvm zfs"
if [ -n "${LXD_CEPH_CLUSTER:-}" ]; then
  storage_backends="${storage_backends} ceph"
fi

for backend in btrfs lvm zfs; do
  if which $backend >/dev/null 2>&1; then
￼    backends="$backends $backend"
  fi
done
```